src/App.tsx

```diff
@@ -21,7 +21,14 @@ export default function App() {
   return (
     <div className="App">
       <h1>Community Focus</h1>
-      <button onClick={() => navigate(`/${slugName}`)}>Join a room</button>
+      <button
+        onClick={() => {
+          navigate(`/${slugName}`);
+          window.location.reload();
+        }}
+      >
+        Join a room
+      </button>
```

Comment:
Lets merge #20 first so you can reformat this to meet the eslint and prettier guidelines.